scripts/azure-windows-release.yml

```diff
@@ -14,9 +14,16 @@ steps:
 
     if ($env:imageName -eq "vs2017-win2016") {
       $env:Path += ";C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin"
+    } elseif ($env:imageName -eq "windows-2019") {
+      $VSCategory = "Enterprise" # alternate 'Community'
+      if (!(Test-Path "C:\Program Files (x86)\Microsoft Visual Studio\2019\${VSCategory}\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin")) {
+        Write-Host "ERROR***: Missing C:\Program Files (x86)\Microsoft Visual Studio\2019\${VSCategory}\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin"
+        $host.SetShouldExit(2)
+      }
+      $env:Path += ";C:\Program Files (x86)\Microsoft Visual Studio\2019\${VSCategory}\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin"
     } else {
       Write-Host "Unknown image name: '$($env:imageName)'"
-      $host.SetShouldExit(1)
```

Comment:
Is this where the mysterious exit code 1 was coming from?

Reply:
I'm not sure, I swapped the exit codes around to hopefully narrow it down but now everything is passing so its no longer reproducible.